src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -531,6 +531,16 @@ CropWindowTool::StatusChangedSignal &CropWindowTool::statusChangedSignal()
 	return m_statusChangedSignal;
 }
 
+Gaffer::Box2fPlug *CropWindowTool::plug() const
+{
+	return m_cropWindowPlug.get();
```

Comment:
Shouldn't we call `findCropWindowPlug()` here prior to accessing `m_cropWindowPlug()`?

Reply:
Added [here](https://github.com/GafferHQ/gaffer/pull/3882/commits/db5594eeaa52688b5b3b1b153684c235c4672e73).